src/screens/lib/responsiveHelpers.js

```diff
@@ -1,3 +1,13 @@
 const getVw = () =>
   Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0);
+
 export const isMobile = () => getVw() < 760;
+
+export const isInViewPort = (element) => {
```

Comment:
Nice work. Did you consider using the Intersection Observer API of modern browsers?

Reply:
didn't heard of it :) but sounds more performant than this solution ... I wonder how much the site slows down because of the scroll event-listener ... is there any way to measure it?

The computer I got  is super slow, so maybe site-performance isn't an issue but just on my machine?!?